lib/bap/bap.mli

```diff
@@ -4262,12 +4262,16 @@ module Std : sig
     (** {2 Backend Interface}  *)
 
     (** [register_backend ~name backend] tries to register backend under
-        the specified [name]. *)
+        the specified [name]. Deprecated, use register_loader instead *)
```

Comment:
other than a comment there should be a deprecation attribute, e.g., `[@@deprecated "user register_loader instead"]`